components/atoms/Label/Label.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
To można wywalić, jest to predefiniowane wyżej w Appie więc tutaj jest niepotrzebne :D

Reply:
takich rzeczy to nie wiedziałem :)